include/dlaf/communication/functions.h

```diff
@@ -0,0 +1,78 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <functional>
+
+#include "dlaf/communication/communicator.h"
+#include "dlaf/communication/mpi_datatypes.h"
+#include "dlaf/mpi_header.h"
+
+namespace dlaf {
+namespace comm {
+
+/// basic wrapper for MPI_Bcast
+template <typename MessageType>
+void bcast(int broadcaster_rank, MessageType& message, Communicator communicator) {
```

Comment:
This is a broadcast of only one element of a type.
Which use do you predict for it?

Reply:
You are right. This PR was just to give a general idea of the API, in the PR #70 you case how I propose to address this aspect.